packages/affiliates/libs/datasets.js

```diff
@@ -204,8 +215,26 @@ function Dataset(basePath, { queries, coingecko, synthPrices }) {
     return path;
   }
 
+  async function saveDappMining(name, config) {
```

Comment:
this saves dataset for dapp mining which omits some data that dev mining needs, such as coingecko and synth data.